### PR TITLE
[Monitor] az monitor metrics alert: refine several help messages

### PR DESCRIPTION
**Pull request by MyronFanQiu**
**Description**  

**Testing Guide**  

**History Notes**  

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.  
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
add to S169

**Comment by yonzhan** [comment]
add to S170

File: src/azure-cli/azure/cli/command_modules/monitor/_help.py [hunk 1]
```diff
@@ -1230,6 +1230,8 @@
 
         Values for METRIC, DIMENSION and appropriate THRESHOLD values can be obtained from `az monitor metrics list-definitions` command.
 
+        Due to server limitation, when an alert rule contains multiple criteria, the use of dimensions is limited to one value per dimension within each criterion.
```
Comment (jsntcy) [inline src/azure-cli/azure/cli/command_modules/monitor/_help.py:1233]: criterias

File: src/azure-cli/azure/cli/command_modules/monitor/_help.py [hunk 2]
```diff
@@ -1239,13 +1241,24 @@
...
             --description "Storage Slow Transactions" \\
             --condition "total transactions > 5 where ResponseType includes Success" \\
-            --condition "avg SuccessE2ELatency > 250 where ApiName includes GetBlob or PutBlob"
+            --condition "avg SuccessE2ELatency > 250 where ApiName includes GetBlob"
+  - name: Create a metric-based alert rule that monitors a custom metric.
+    text: |
+        az monitor metrics alert create -n "metric alert rule on a custom metric" -g "Demos" --scopes {VirtualMachineID} \\
+            --condition "max Azure.VM.Windows.GuestMetrics.Memory\\Available Bytes > 90" \\
+            --window-size 5m --evaluation-frequency 1m
+  - name: Create a high CPU usage alert on several VMs with no actions.
```
Comment (jsntcy) [inline src/azure-cli/azure/cli/command_modules/monitor/_help.py:1257]: no action

File: src/azure-cli/azure/cli/command_modules/monitor/_help.py [hunk 3]
```diff
@@ -1293,6 +1306,8 @@
 
         Values for METRIC, DIMENSION and appropriate THRESHOLD values can be obtained from `az monitor metrics list-definitions` command.
 
+        Due to server limitation, when an alert rule contains multiple criteria, the use of dimensions is limited to one value per dimension within each criterion.
```
Comment (jsntcy) [inline src/azure-cli/azure/cli/command_modules/monitor/_help.py:1309]: criterias

File: src/azure-cli/azure/cli/command_modules/monitor/_help.py (hunk 1 above)
Comment (jsntcy) [inline src/azure-cli/azure/cli/command_modules/monitor/_help.py:1233]: not consistent: criterion vs criteria

**Pull request merged**